e2e/testconfig/testconfig.go

```diff
@@ -453,13 +455,13 @@ func defaultGovv1ModifyGenesis() func(ibc.ChainConfig, []byte) ([]byte, error) {
 func defaultGovv1Beta1ModifyGenesis() func(ibc.ChainConfig, []byte) ([]byte, error) {
 	const appStateKey = "app_state"
 	return func(chainConfig ibc.ChainConfig, genbz []byte) ([]byte, error) {
-		genesisDocMap := map[string]interface{}{}
+		genesisDocMap := map[string]any{}
```

Comment:
similar to Cians other comment, I'm generally in favor of `any` due to it being more succinct and commonplace in other langs but don't mind either way.

Reply:
K, I'll keep use-any enabled.